CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ This changelog keeps track of all changes to the packs-sdk. We follow convention
 - **Breaking Change** Added a validation rule that prevents the usage of varargsParameters for sync table getters
   which are not currently supported in the UI.
 - **Future Breaking Change** The properties `id`, `primary`, and `featured` of object schemas will be renamed to `idProperty`, `displayProperty`, and `featuredProperties`, respectively, to better clarify that their values refer to property names within the schema. During a migration period, you may use either name; the original names are now marked as deprecated and will be removed at a future date, no earlier than May 1, 2022.
+- Added "force" option to `LinkSchema` for `LinkDisplayType.Embed`.
+- **Breaking Change** Removed dead code related to `defaultConnectionType`. This was an unused feature and should not have been previously referenced.
```

Comment:
I think these are redundantly included, likely a merge conflict with Spencer moving these to a separate section below.

Reply:
Oops! Thanks, done. I'll merge this when my coda repo change is ready to merge, too.